tests/custom_maven_install/regression_testing_install.json

```diff
@@ -2175,34 +2175,34 @@
                 "url": "https://repo1.maven.org/maven2/com/microsoft/azure/adal4j/1.6.0/adal4j-1.6.0.jar"
             },
             {
-                "coord": "com.nimbusds:lang-tag:1.6",
+                "coord": "com.nimbusds:lang-tag:1.7",
```

Comment:
FYI repinning just did this upgrade to 1.7. I can add another version set to fix if preferred.